phototherapy/js/phototherapy.spec.js

```diff
@@ -1,3 +1,5 @@
+// Do we still need this file?
```

Comment:
@talavinari @yonatankatz 
This test is angularJs specific, and it breaks on master.
Do we still need it or can I remove it?

Reply:
And do we need `karma` for testing?
`jest` is much more popular and well maintained, `karma` is kinda dead :/

Reply:
totally don't need it. it is our first try to kick off the E2E testing. we can safely delete the spec file and the karma package

Reply:
removed 'karma'